docs/plugin-sftp.md

```diff
@@ -142,7 +142,24 @@ default value of 60 minutes is used.
 Shesmu tries to use GNU findutils to explore the remote directory. This is
 convenient because it is standard with Linux. However, if file have names which
 are not allowed in JSON strings, it falls apart rather quickly. If this is the
-case, use the [JSON directory listing tool](../json-dir-list) to ensure the
-output is always correctly encoded. Install it on the target system and then
-set `"listCommand"` to the path to the program or just `"json-dir-list"` if
-it's installed into a location on the `PATH`.
+case, use the JSON directory listing tool included in the Shesmu repository to
+ensure the output is always correctly encoded. Install it on the target system
+and then set `"listCommand"` to the path to the program or just
+`"json-dir-list"` if it's installed into a location on the `PATH`.
+
+## JSON Directory Listing Tool
+For the `unix_file` input format, Shesmu will try to crawl the directory
+structure using a `find` command to produce JSON output. This works about as
+well as one could hope. If file names, user names, or group names have
+characters that are now allowed in JSON strings, it does not go well. This tool
+provides a more robust alternative that does the JSON encoding correctly. To
+install it:
+
+    sudo apt-get install build-essential pkg-config libjsoncpp-dev
+    autoreconf -i
+    ./configure --prefix=/install/dir
+    make
+    make install
```

Comment:
Why does this tool not get an automatic build like the others?

Reply:
Good idea! I will add.

I don't know about building Shesmu releases this way. I've never had success with the Maven repository GitHub provides. We could have to push to the OICR Artifactory with the right keys.